crates/router/src/connector/adyenplatform/transformers/payouts.rs

```diff
@@ -235,7 +235,7 @@ impl<F> TryFrom<&types::PayoutsRouterData<F>> for AdyenTransferRequest {
                 let payout_type = request.get_payout_type()?;
                 Ok(Self {
                     amount: adyen::Amount {
-                        value: request.amount,
+                        value: request.minor_amount,
```

Comment:
again, please convert and pass and donot use, the amount directly if passed from core